tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -942,8 +990,14 @@ async function checkInClusterLegacyEvent(targetNamespace, testSubscriptionV1Alph
   }
 
   const eventData = {'id': eventId, 'legacyOrder': '987'};
-  const eventType = testSubscriptionV1Alpha2? eventTypeOrderCompleted.replace('.v1', ''): '';
-  const eventSource = testSubscriptionV1Alpha2? eventSourceInApp: '';
+  if (!eventType) {
+    eventType = testSubscriptionV1Alpha2 ? eventTypeOrderCompleted.replace('.v1', '') : '';
+  } else {
+    eventType = testSubscriptionV1Alpha2 ? uncleanEventType.replace('.v1', '') : '';
+  }
+  if (!eventSource) {
+    eventSource = testSubscriptionV1Alpha2 ? eventSourceInApp : '';
+  }
```

Comment:
same, We can also move these if blocks to `checkInClusterEventDelivery(targetNamespace, testSubscriptionV1Alpha2=false)` so everything related to eventtypes are in one place for `v1alpha2`.

Reply:
Unfortunately, it is handled differently for legacy events. Otherwise, yes it'd be good idea to move it.